config/cucumber.yml

```diff
@@ -0,0 +1,9 @@
+<%
+rerun = File.file?('rerun.txt') ? IO.read('rerun.txt') : ""
```

Comment:
should these be single quotes?

Reply:
It's scaffolded yml I will not touch that :)

Reply:
okay